compiler-plugin/testData/api/throwList.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Google LLC
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+// TEST PROCESSOR: ThrowListProcessor
+// EXPECTED:
+// java.io.IOException
+// java.io.IOException,java.lang.IndexOutOfBoundsException
+// END
+// FILE: ThrowsException.java
```

Comment:
can we add a test with library so that we ensure the descriptor version works as well?

Reply:
descriptor version won't work unless we read bytecode directly. Throw list does not have a place in kotlin compiler descriptors.

Reply:
Sounds like a good comment in the API :)

Reply:
ugh that hurts :/.
we should document that behavior as it might matter a lot when generating java code that overrides descriptors coming from compiled code :/. Though I'm not even sure how developer can solve that kind of issue.